xml/signals/SPTCO-1930/appearance-DW-2-head.xml

```diff
@@ -114,22 +114,27 @@
     <aspectMapping>
       <advancedAspect>Diverging Clear</advancedAspect>
       <ourAspect>Clear</ourAspect>
+      <ourAspect>Diverging Clear</ourAspect>
```

Comment:
I think this addition means that the signal in advance of a Diverging Clear is another Diverging Clear.

Is that right?

My understanding of "Diverging Clear" is "expect to take the diverging route at the _next_ signal; clear speed authorized" (emphasis added)